ONSdigital/eq-survey-runner
# EQ-14 Front-end Framework

## Changes
- Added basic Gulp setup including Sass/ES6 compiling, minification, linting, etc.
- Added Karma test runner with Mocha framework
- Added single test 
- Created basic base template
- Added basic Sass framework with grid (WIP)
## How to test
- install npm dependencies with `npm install`
- wait 2 hours while they install
- run `npm run build`, this should build assets into 'static' folder
- confirm this works in your browser
- run `npm run tests`
- confirm single test runs successfully
